### PR TITLE
with slider but not with rows

### PR DESCRIPTION
**Pull request by subhadeepchowdhury41**

### CODE REVIEW

**Comment by akshitadixit** [comment]
@subhadeepchowdhury41 please write better and clearer commit messages next time

**Comment by Suryanshomar7240** [comment]
@subhadeepchowdhury41 always try to run "npm run build" before committing also please solve this.
bug

**Comment by yashraj-02-techie** [comment]
Resolve the build errors

File: src/components/projects/allProjects/Projects.js [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
-import { FaGithub, FaArrowRight, FaArrowDown } from "react-icons/fa";
-import {FiMail} from "react-icons/fi"
+import {
+  FaArrowRight,
+  FaArrowLeft,
+} from "react-icons/fa";
+import "react-icons/fi";
 
 const Projects = () => {
   let [proj_type, setproj_type] = useState("all");
```
Comment (yashraj-02-techie) [inline src/components/projects/allProjects/Projects.js:16]: Remove unused variables.

**Review by yashraj-02-techie — CHANGES_REQUESTED** [review]

**Comment by yashraj-02-techie** [comment]
Resolve the conflicts and commit the changes

**Pull request closed without merging**